inc/components/components/post-featured-image/component.php

```diff
@@ -28,15 +28,20 @@
 
 			$thumbnail_id = get_post_thumbnail_id( $post_id );
 
-			return array_merge(
-				$config,
-				[
-					'alt'     => get_image_alt( $thumbnail_id ),
-					'caption' => wp_get_attachment_caption( $thumbnail_id ),
-					'credit'  => get_post_meta( $thumbnail_id, 'credit', true ),
-					'src'     => get_the_post_thumbnail_url( $post_id ),
-				]
-			);
+			$config['caption'] = wp_get_attachment_caption( $thumbnail_id );
+			$config['credit']  = get_post_meta( $thumbnail_id, 'credit', true );
```

Comment:
Note: I think we should probably support caption/credit for the image component as well and move this to the `get_image_component_attributes()` function, but thought that was out of scope for this issue.